gradlew.bat

```diff
@@ -43,11 +43,11 @@ set JAVA_EXE=java.exe
 %JAVA_EXE% -version >NUL 2>&1
 if %ERRORLEVEL% equ 0 goto execute
 
-echo.
-echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
+echo. 1>&2
+echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH. 1>&2
+echo. 1>&2
+echo Please set the JAVA_HOME variable in your environment to match the 1>&2
+echo location of your Java installation. 1>&2

```

Comment:
These kinds of "redirected" error blocks often read better as a quick function:
```suggestion
write_java_home_error() {
  echo
  echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
  echo
  echo Please set the JAVA_HOME variable in your environment to match the
  echo location of your Java installation.
}
write_java_home_error 1>&2
```

Reply:
This isn't optimization, it's clarity and error reduction.
Also, perhaps don't push a PR if the response to everything will be "I'm not changing it"...

Reply:
It's not optimization, just readability and less error-prone when editing...